docs/use-cases/airdrop.mdx

```diff
@@ -1,10 +1,18 @@
 ---
 slug: /use-cases/passport-enabled-airdrops
+title: Passport-enabled airdrops
 description: Privacy-preserving proof of personhood, anti bot and sybil-resistant protection for airdrops using biometric passports and zero-knowledge proofs.
 ---
 import OutLink from "@site/src/components/OutLink";
 import IdealImage from '@site/src/components/IdealImage';
 
+<head>
```

Comment:
You can just add `image: img/airdrops-og-img.png` near the title and description

Reply:
@ihordiachenko fixed, thanks